### ISSUE TITLE
Jack on windows: my experience 

### ISSUE THREAD

**Issue by kjbf** [jackaudio/jack2#601]
I know this is not the right place for this kind of thing but I don't think there is any other place express this thoughts. 

I've been using jack on windows for sometimes and here are the problems I encountered.
1. cannot start jack from qjackctl, but if I give the same command from the command line it works;
    * When I installed jack this problems didn't exist but, for some reason, it now exists and no matter how many time I try to reinstall jack it persists.
2. cannot use wasapi;
3. midi is kinda of strange;
    * there are no names — it is not that bad but something like a2jmidid would be wounderful
    * There is extreme latency with non class compliant midi controller:
 I have 2 keyboards a m-audio code 49 (class compliant) and a korg SV-1 (non class compliant). the code 49 works normally, instead the midi received through the SV-1 has extreme latency added — if connected through USB. 
    If I connected through the audio interface the SV-1 has no latency. On top of that If I route the usb-connected sv-1 through a virtual midi port (loopmidi) and then the virtual midi port to ardour (carla or musescore) the latency disappear — there is still some but it is added by loopmidi. May it be a problem with non class compliant  midi keyboard? 
    (the audio interface is a class compliant steinberg)

In conclusion, I'm not complaining, I know it is a lot of work and these are many features requests. I thank you in the first place for all this, just wanted to share my experience.
